spring-kafka-docs/src/main/asciidoc/retrytopic.adoc

```diff
@@ -429,6 +429,50 @@ public RetryTopicConfiguration myRetryTopic(KafkaTemplate<String, MyPojo> templa
 
 NOTE: The default behavior is to suffix with the delay values, except for fixed delay configurations with multiple topics, in which case the topics are suffixed with the topic's index.
 
+===== Custom naming strategies
+
+More complex naming strategies can be accomplished by registering a bean that implements `RetryTopicNamesProviderFactory`. The default implementation is `SuffixingRetryTopicNamesProviderFactory` and a different implementation can be registered in the following way:
+
+====
+[source, java]
+----
+@Bean
+public RetryTopicNamesProviderFactory myRetryNamingProviderFactory() {
+    return new CustomRetryTopicNamesProviderFactory();
+}
+----
+====
+
+As an example the following implementation, in addition to the standard suffix, add a prefix to retry/dl topics names:
```

Comment:
I think it should be "adds a prefix", shouldn't it?

Reply:
Yes, thank you 👍